CHANGES.md

```diff
@@ -5,6 +5,16 @@ wavesurfer.js changelog
 ------------------
 
 - New `barRadius` option to create waveforms with rounded bars (#953)
+- New `WebAudioMedia` backend (#1767):
+  - It allows to use API Web Audio also with big audio files, loading audio like with MediaElement backend (HTML5 audio tag), 
+    so you have to use the same methods of MediaElement backend for loading and playback. In this way, the audio
+    resource is not loaded entirely from server, but in ranges, allowing you to use WebAudio features, like filters, 
+    also with audio with big duration. You have to give also peaks, so the audio data are not decoded.
+    For example:
+    ` wavesurfer.load(url, peaks, preload, duration);
+      wavesurfer.play();
+      wavesurfer.setFilter(customFilter);
+    `
```

Comment:
This example is not good? The same in changelog

Reply:
I was thinking of an example in the `example` directory that uses the new backend (and adds some filters for example).

Reply:
I've pushed the example.
- The filters provided are good? In the explanation i've commented only one because took me a lot of space.
- I've uploaded a file audio taken from `split-channels` example. I've initially wanted to load a 3 hours audio (because with an audio of this dimensions, with `WebAudio` backend the browser crashes, but with `MediaElementWebAudio` not, but GitHub tell me that the file is too large).
- I've used `audiowaveform` with `split-channels` command, but the data array that returned is only of one dimension, so i drawn the entire waveform, without splitting the channels (if you know how i can solve the problem, tell me)